sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/SchemaRegistryClient.java

```diff
@@ -36,51 +36,96 @@
 
 
 /**
- * HTTP-based client that interacts with Azure Schema Registry service to store and retrieve schemas on demand.
+ * {@link SchemaRegistryClient} is an HTTP-based client that interacts with Azure Schema Registry service to store and
+ * retrieve schemas on demand.
  *
- * <p><strong>Register a schema</strong></p>
- * Registering a schema returns a unique schema id that can be used to quickly associate payloads with that schema.
- * <!-- src_embed com.azure.data.schemaregistry.schemaregistryclient.registerschema -->
+ * <p><strong>Sample: Construct a {@link SchemaRegistryClient}</strong></p>
+ *
+ * <p>The following code sample demonstrates the creation of the synchronous client
+ * {@link com.azure.data.schemaregistry.SchemaRegistryClient}.  The credential used is {@code DefaultAzureCredential}
```

Comment:
We can copy the auth data from package-info in above and add more details about fully-qualified-namespace.